packages/react-native-reanimated/src/Bezier.ts

```diff
@@ -7,6 +8,8 @@
 
 // These values are established by empiricism with tests (tradeoff: performance VS precision)
 
+const logger = global.__reanimatedLogger;
```

Comment:
I had to access `__reanimatedLogger` via `global` as next example was not seeing `__reanimatedLogger` when I accessed it directly, thus it showed this error:

```
ReferenceError: __reanimatedLogger is not defined
```

Reply:
Have you tried `globalThis` instead?